pkg/fanal/analyzer/language/nodejs/license_test.go

```diff
@@ -0,0 +1,125 @@
+package nodejs
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/nodejs/packagejson"
+	"github.com/aquasecurity/trivy/pkg/mapfs"
+)
+
+func Test_ParseLicenses(t *testing.T) {
+	tests := []struct {
+		name    string
+		dir     string
+		want    map[string][]string
+		wantErr string
+	}{
+		{
+			name: "happy",
+			dir:  filepath.Join("testdata", "happy"),
+			want: map[string][]string{
```

Comment:
Can you add a test case with a missing license?
I am referring to case where license field in `package.json` is missing and `LICENSE` file doesn't exist.

Reply:
done